common/data/data.json

```diff
@@ -988,14 +988,14 @@
       "sell_price": 1024,
       "rarity": 50000,
       "sticky": false,
-      "tags": ["mine", "fish", "mkwii"]
```

Comment:
Why remove this from being fishable?

Reply:
So people can't just fish for both high rarities

Reply:
ok